Tests/FTPropertyWrappersTests/XCTestManifests.swift

```diff
@@ -5,8 +5,6 @@ public func allTests() -> [XCTestCaseEntry] {
     return [
         testCase(KeychainTests.allTests),
         testCase(SerializedTests.allTests),
-        testCase(UserDefaultsTests.allTests),
-        testCase(StoredSubjectTests.allTests),
```

Comment:
Jestli jsem dobre pochopila, tak se zmenil pristup, jak k temto objektu pristupovat -> nemely by se i presto nejake testy zachovat/napsat nove?

Reply:
Na systémové věci, které jsou mimo scope této package úplně nevím jaké testy bychom psali.